index2.html

```diff
@@ -0,0 +1,360 @@
+          </div>
+          <div class="top-row">
+            <ul>
+              <li class="boot">Bootstrap</li>
+              <li class="ruby-rails">Ruby on rails</li>
+              <li class="r-spec">RSpec</li>
+            </ul>
+          </div>
+
+          <div class="bottom-row">
+            <ul>
+              <li class="Capybara">Capybara</li>
+              <li class="Sele">Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="language-container">
+          <div class="language-column">
+            <h1 class="skills">Skills</h1>
+            <img class="light" alt="" src="lighting.svg" />
+          </div>
+          <div class="top-row">
+            <ul>
+              <li class="codekick">Codekick</li>
+              <li class="github">Github</li>
+              <li class="codepen">Codepen</li>
+              <li class="gitlab">Gitlab</li>
+            </ul>
+          </div>
+          <div class="bottom-row">
+            <ul>
+              <li class="term">Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer class="contactpage">
+      <section class="projects">
+        <h2 class="textline1">I'm always</h2>
+        <h2 class="textline2">interested in</h2>
+        <h2 class="textline3">hearing about</h2>
+        <h2 class="textline4">new projects, so</h2>
+        <h2 class="textline5">if you'd like to</h2>
+        <h2 class="textline6">chat please get</h2>
+        <h2 class="textline7">in touch</h2>
+      </section>
+      <section class="projects-2-big">
+        <div class="new-paragraph">
+          I'm always interested in hearing <br />about new projects, so if you'd
+          <br />
+          like to chat please get in touch.
+        </div>
+      </section>
+
+      <section class="form-section">
+        <form action="https://formspree.io/f/myyvzkag" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                maxlength="30"
+                name="user_name"
+                class="name-text-box"
+                id="full-name"
+                placeholder="Full Name"
+                required=""
+              />
+            </li>
+
+            <li>
+              <input
+                type="email"
+                name="user_email"
+                class="name-last-text-box"
+                id="email"
+                placeholder="Email Address"
+                required=""
+              />
+            </li>
+            <li>
+              <textarea
+                id="text-box"
+                name="message"
+                maxlength="500"
+                class="enter-form"
+                cols="30"
+                rows="10"
+                placeholder="Enter text here"
+                required=""
+              ></textarea>
+            </li>
+          </ul>
+
+          <button type="submit" class="touch-btn">Get in touch</button>
```

Comment:
- [ ] Great job 👍 . Kindly fix the color contrast issue of your "Get in Touch" `button` located in your Contact Me section 🙏

![image](https://user-images.githubusercontent.com/74814780/188692838-651ad95c-63c0-4871-9cee-154fe3325ffd.png)